tasks/main.yml

```diff
@@ -25,11 +25,19 @@
 - name: enable passwordless sudo for assorted groups
   become: yes
   become_user: root
-  template: >
-    src={{item}}.j2
-    dest={{sudoers_dropin_dir}}/{{item}}
+  template:
+    src: "{{item}}.j2"
+    dest: "{{sudoers_dropin_dir}}/{{item}}"
+    mode: "{{ sudoers_dropin_mode }}"
   with_items:
     - nopasswd
     - proxyenv
-    
+
+- name: Ensure sudoers dropin directory is included
```

Comment:
I've had some old RHEL5 nodes, that didn't have it for sure. Based on this [changelog](https://www.rpmfind.net/linux/RPM/centos/updates/6.8/x86_64/Packages/sudo-1.8.6p3-25.el6_8.x86_64.html), it was added about 6 years ago:

    * Wed Mar 23 2011 Daniel Kopecek <dkopecek@redhat.com> - 1.7.4p5-4
      - added #includedir /etc/sudoers.d to sudoers
        Resolves: rhbz#615087

So this might be only needed for old systems, but it's also possible that the line was removed manually so I guess it's better not no rely on its presence. (especially when sudoers.d wasn't even created on RHEL5 by default, so changing `/etc/sudoers` directly was more common)